frontend/support/sortfilterproxymodel.cpp

```diff
@@ -38,7 +38,7 @@
 **
 ****************************************************************************/
 
-#include "sortfilterproxymodel.h"
+#include "sortfilterproxymodel.hpp"
```

Comment:
@enervey does this address the issue for you? it's a different file to the one you referenced but popped up as an error for me while trying a full rebuild.